client/dashboard/components/stat/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { __experimentalHStack as HStack, ProgressBar } from '@wordpress/components';
+import './style.scss';
+
+interface StatProps {
+	density?: 'low' | 'high';
+	description?: string;
+	descriptionAlignment?: 'start' | 'end';
```

Comment:
I used "start" and "end" to align with CSS logical position names. More RTL friendly that using "left" and "right".